quipucords/tests/api/source/test_source.py

```diff
@@ -2090,13 +2091,8 @@ def test_create_invalid_hosts(self, client_logged_in, net_cred):
         """Test invalid host patterns."""
         hosts = [
             "192.1..2",
-            "192.01.5.10",
             "192.1.5.1/",
```

Comment:
Some hosts that were considered invalid in the previous implementation are now technically valid. Even if they are not valid IPs, they might be considered valid according to the FQDN hostname validation rules.